### PR TITLE
Enable session security

### PR DESCRIPTION
**Pull request by joecohens**
Using the new Laravel 5.1 cipher :)

/cc @cachethq/owners

### CODE REVIEW

File: config/session.php [hunk 1]
```diff
@@ -53,7 +53,7 @@
     |
     */
 
-    'encrypt' => false,
+    'encrypt' => true,
```
Comment (GrahamCampbell) [inline config/session.php]: Are you sure?

Comment (GrahamCampbell) [inline config/session.php]: This will break all ajax requests won't it?

Comment (joecohens) [inline config/session.php]: Not sure at all lol I just saw you both change it, thought it was because the security issue :stuck_out_tongue_winking_eye:

Comment (joecohens) [inline config/session.php]: Reverted

**Comment by GrahamCampbell** [comment]
Could you run composer update again plz?

**Comment by joecohens** [comment]
Yep.. Squashing after btw

**Comment by joecohens** [comment]
No changes lol

**Comment by GrahamCampbell** [comment]
> No changes lol

Oh yeh, the time in the lock file isn't BST. Forgot my timezone is +1 atm, lol/

**Comment by joecohens** [comment]
Lol

**Pull request merged**